app/population/view/daily.njk

```diff
@@ -0,0 +1,116 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
+{% from "data/macro.njk" import appData %}
+
+{%  macro statsEntry(property, value) %}
+  {%  set key = "population::daily_view.labels." + property %}
+  {%  set label = t(key) %}
+
+  <h4 class="govuk-heading-s">{{ label }}</h4>
+  <p class="govuk-body">{{ value }}</p>
+{% endmacro %}
+
+{% macro statsGroup(headingProperty, stats) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  {%  for stat in stats %}
+    {{  statsEntry(stat.property, stat.value) }}
+  {% endfor %}
+{% endmacro %}
+
+{% macro transfersGroup(headingProperty, transfers) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  <p class="govuk-body">
+  {%  for transfer in transfers %}
+    {{ transfer.count }} from {{  transfer.establishment }}<br>
+  {% endfor %}
+  </p>
+{% endmacro %}
+
+{% set displayDate = details.date if details.date else date | formatDateWithRelativeDay %}
+
+{% block pageTitle %}
+  {{ t("population::daily_view.page_title_with_date", {
+    date: displayDate
+  }) }}
+{% endblock %}
+
+
+{% block content %}
+  <header class="govuk-grid-row govuk-!-margin-bottom-7">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
+        <span class="govuk-caption-xl">
+            {{ t("population::daily_view.subheading") }}
+            </span>
+        <span class="govuk-!-display-block">{{ displayDate }}</span>
+      </h1>
+    </div>
+  </header>
+
+  {%  call appPanel({classes: 'govuk-!-margin-bottom-4'}) %}
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+
+        {% if spaces.details.updated_at %}
+          <h2 class="govuk-heading-xl">{{ t("population::daily_view.free_spaces_with_count", {count:spaces.details.free_spaces}) }}</h2>
+          <p class="govuk-body">{{ t("population::daily_view.free_space_description") }}</p>
+          <p class="govuk-body"><b>{{  t("population::daily_view.labels.last_updated") }} {{  spaces.details.updated_at | formatDateWithTimeAndDay }}</b></p>
+          <div>
+            <a href="" class="govuk-link">
+              <strong>{{ t("population::daily_view.actions.change_numbers", {
+                context: "with_visually_hidden_text"
+              }) | safe }}</strong>
+            </a>
+          </div>
+        {% endif %}
+
+        {%  if not spaces.details.updated_at %}
```

Comment:
Any reason this can't be an `else` instead of 2 separate `if` blocks?

Reply:
Replaced with an `else` block